### DISCUSSION TITLE
Change the name of the application

### DISCUSSION THREAD

**Discussion by Frigyes06** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#54] [Polls]
I'm proposing to change the name of this application from Universal Android Debloater (UAD) to Universal Android Debloater Next Generation (UAD-ng).
This would help people differentiate between the old software and the new one. The rename would also make it easier to package the software for package managers such as the AUR, APT, flatpak, etc.

**Comment by lavafroth** [comment]
Distillery
